### DISCUSSION TITLE
How to make comments posted on issues use "github-actions[bot]" as username

### DISCUSSION THREAD

**Discussion by Oren-Lindsey** [community/community#25261] [Actions]
<p>I have an action to post on issues made in my repositories. I generated a personal access token in my account settings and used that for authentication. And whenever it posts, it posts as my account. I don’t want that to happen. I realized it was because the token is for my account, but I can’t figure out another way. My code is <a href="https://github.com/Oren-Lindsey/wasteof-client/blob/759a371f37807c6d34f5df2b9b260fe688126d06/.github/workflows/github-actions.yml" rel="noopener nofollow ugc">here</a>. Pls help</p>

**Comment by kingthorin** [comment, marked as answer]
<p>Just pass the default GitHub token instead.</p><aside class="onebox allowlistedgeneric" data-onebox-src="https://docs.github.com/en/actions/security-guides/automatic-token-authentication">
  <header class="source">
      <img src="https://docs.github.com/assets/cb-803/images/site/favicon.svg" class="site-icon" width="16" height="16">

      <a href="https://docs.github.com/en/actions/security-guides/automatic-token-authentication" target="_blank" rel="noopener nofollow ugc">GitHub Docs</a>
  </header>

  <article class="onebox-body">
    <img src="https://user-images.githubusercontent.com/7570458/181089944-c0d390b8-89d4-496c-a213-d2a30bf7ccc7.png" class="thumbnail onebox-avatar" width="500" height="500" srcset="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_500x500.png, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_750x750.png 1.5x, https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_1000x1000.png 2x" data-small-upload="https://aws1.discourse-cdn.com/github/optimized/2X/3/337b7cc06184984e148c075df20ef009e0e6ecaa_2_10x10.png">

<h3><a href="https://docs.github.com/en/actions/security-guides/automatic-token-authentication" target="_blank" rel="noopener nofollow ugc">Automatic token authentication - GitHub Docs</a></h3>

  <p>GitHub provides a token that you can use to authenticate on behalf of GitHub Actions.</p>


  </article>

  <div class="onebox-metadata">
    
    
  </div>

  <div style="clear: both"></div>
</aside>
